packages/api/src/external/commonwell/patient-conversion.ts

```diff
@@ -47,6 +47,11 @@ export function patientToCommonwell({
     assigner: orgName,
   };
   const strongIds = getCwStrongIdsFromPatient(patient);
+  const cwGender = mapGenderAtBirthToCw(patient.data.genderAtBirth);
+  if (!cwGender)
+    throw new MetriportError("Missing gender on patientToCommonwell", undefined, {
```

Comment:
i think this error will get caught wherever ``setupApiAndCwPatient` is called, and we will get sentry alerts for every time this happens?

Reply:
No, this should not ever execute, since CW will be disabled for a patient with gender `U`, which is what can lead to `mapGenderAtBirthToCw` to return `undefined`.

I see how there's an interdependency across `mapGenderAtBirthToCw` and `isCommonwellEnabledForPatient`, but I think this is fine for this being a patch and I want to minimize changes in the codebase.

Cool?